lib/index.js

```diff
@@ -30,10 +31,17 @@ Outbound.prototype.initialize = function() {
   window.outbound.methods = [
     'identify',
     'track',
+    'alias',
     'registerApnsToken',
     'registerGcmToken',
     'disableApnsToken',
     'disableGcmToken',
+    'disableAllGcmTokens',
```

Comment:
Yes it should be fixed - looks like some errors on IE 9? https://circleci.com/gh/segment-integrations/analytics.js-integration-outbound/314

I'll verify this isn't on our side.